ShareTheTable/View Controllers/Profile Making/Question/QuestionViewController.m

```diff
@@ -10,13 +10,110 @@
 #import "SceneDelegate.h"
 #import "FeedViewController.h"
 #import "Parse/Parse.h"
+#import "User.h"
 
 @interface QuestionViewController ()
 
 @end
 
 @implementation QuestionViewController
 
+- (IBAction)didTapAButton:(id)sender {
```

Comment:
These are all changes from the previous pull request right? We don't want old changes in newer pull requests, I'm not sure why this is happening, but if you merge a PR right after creating it, then rebase your work on top of main again, when you create a second PR you won't have old changes. 

This is not intuitive because you'd think you'd wait for your code to be approved before merging it, but because you're working solo, this is fine. You can create another PR to fix/address feedback afterwards.